src/icons/check-in.svg

```diff
@@ -0,0 +1,3 @@
+<svg>
+<path d="M18.0002 6L12.0002 12L9.0002 9M15.3 4C14.3 3.3 13.2 3 12 3C8.7 3 6 5.7 6 9C6 11.5 6.7 13 8.3 15.4C9.4 17 10.7 18.5 12 20C13.3 18.6 14.5 17 15.7 15.4C17 13.8 17.8 12 18 10" stroke="#0074C8" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
This won't work. We cannot use a stroke or a specific stroke color. 

The SVG must be a solid shape.